build/assets/scripts/pre-boot-tuning.sh

```diff
@@ -2,36 +2,22 @@
 
 set -euo pipefail
 
-non_iso_cpumask=""
+non_iso_cpumask_invert="" #${NON_ISOLATED_CPU_MASK_INVERT}
 cpu_affinity=""
 
+SYSTEM_CONFIG_FILE="/etc/systemd/system.conf.d/setAffinity.conf"
+
 get_reserved_cores() {
```

Comment:
This function can probably go away as well, right?

Reply:
yes..  it here for now as a backup incase i see that cpuaffinity setup isnt really working via ignition (backup code is commented out right now and will be removed once verified)

Reply:
so can we remove this method?